### PR TITLE
Rohini pilla

### PR DESCRIPTION
**Pull request by Rohinipilla**
I changed name to Rohini Pilla

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 var msgs = [
-  "Sagar Pawar is learning Git 😄",
```
Comment (sagarpawardev) [inline script.js:2]: Why this change?

Comment (Rohinipilla) [reply script.js:2]: i just want to see my name that's all

Comment (sagarpawardev) [reply script.js:2]: Ok Cool!

**Review by sagarpawardev — APPROVED** [review]
